src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -95,9 +219,37 @@ public override int GetHashCode()
             var hashCodeCombiner = HashCodeCombiner.Start();
             hashCodeCombiner.Add(Name, StringComparer.Ordinal);
             hashCodeCombiner.Add(Value);
```

Comment:
Minor but since `Equals` ignores `Value` if `Structure == HtmlAttributeStructure.Minimized`, so should `GetHashCode()`.